app/views/appointments/edit.html.erb

```diff
@@ -3,8 +3,22 @@
   { title: "Edit appointment for #{@appointment.name}" }
 ) %>
 
-<h1>Edit appointment for <%= @appointment.name %> <span class="text-muted"><span class="sr-only">Booking reference </span>#<%= @appointment.id %></span></h1>
-<%= rebooked_from_heading(@appointment) %>
+<div class="row">
+  <div class="col-md-10">
+    <h1>Edit appointment for <%= @appointment.name %> <span class="text-muted"><span class="sr-only">Booking reference </span>#<%= @appointment.id %></span></h1>
+    <%= rebooked_from_heading(@appointment) %>
+  </div>
+  <div class="col-md-2 action-buttons">
+    <%= link_to appointment_reschedule_path(@appointment), title: 'Reschedule appointment', class: 'btn btn-info' do %>
+      <span class="glyphicon glyphicon-calendar" aria-hidden="true"></span>
+      <span class="sr-only">Reschedule appointment</span>
```

Comment:
I'd be half tempted to show the full label of these buttons on this screen. Might be worth a try if it doesn't break the layout too badly?